experimental/datacommons-js/example.html

```diff
@@ -66,5 +66,32 @@ <h3>Line</h3>
       place="country/USA"
       variables='["Count_Person"]'
     ></datacommons-line>
+    <h2>Lollipop</h2>
+    <datacommons-bar
+      title="Population in States of United States"
+      place="country/USA"
+      childPlaceType="State"
+      variable="Count_Person"
+      lollipop="true"
```

Comment:
you can drop the `="true"` part here- and in fact `="false"` i dont think will work as expected. Just the presence of the `lollipop` attribute with no value should set the flag

Reply:
Done!